antenna/breakpad_resource.py

```diff
@@ -267,70 +265,16 @@ def extract_payload(self, req):
         return raw_crash, dumps
 
     def get_throttle_result(self, raw_crash):
-        """Given a raw_crash, figures out the throttling
-
-        If the raw_crash contains throttling information already, it returns
-        that. If it doesn't, then this will apply throttling and return the
-        results of that.
-
-        A rule name of ``ALREADY_THROTTLED`` indicates that the raw_crash was
-        previously throttled and we're re-using that data.
-
-        A rule name of ``THROTTLEABLE_0`` indicates that the raw_crash was
-        marked to not be throttled.
+        """Runs raw_crash through throttler for a throttling result
 
         :arg dict raw_crash: the raw crash to throttle
 
         :returns tuple: ``(result, rule_name, percentage)``
 
         """
-        # FIXME(willkg): This is commented out so we can test throttling on
-        # stage for bug #1476941.
-
-        # # If the raw_crash has a uuid, then that implies throttling, so return
-        # # that.
-        # if 'uuid' in raw_crash:
-        #     crash_id = raw_crash['uuid']
-        #     if crash_id[-7] in (str(ACCEPT), str(DEFER)):
-        #         result = int(crash_id[-7])
-        #         throttle_rate = 100
-
-        #         # Save the results in the raw_crash itself
-        #         raw_crash['legacy_processing'] = result
-        #         raw_crash['throttle_rate'] = throttle_rate
-
-        #         return result, 'FROM_CRASHID', throttle_rate
-
-        # # If we have throttle results for this crash, return those.
-        # if 'legacy_processing' in raw_crash and 'throttle_rate' in raw_crash:
-        #     try:
-        #         result = int(raw_crash['legacy_processing'])
-        #         if result not in (ACCEPT, DEFER):
-        #             raise ValueError('Result is not a valid value: %r', result)
-
-        #         throttle_rate = int(raw_crash['throttle_rate'])
-        #         if not (0 <= throttle_rate <= 100):
-        #             raise ValueError('Throttle rate is not a valid value: %r', result)
-        #         return result, 'ALREADY_THROTTLED', throttle_rate
-
-        #     except ValueError:
-        #         # If we've gotten a ValueError, it means one or both of the
-        #         # values is bad and we should ignore it and move forward.
-        #         mymetrics.incr('throttle.bad_throttle_values')
```

Comment:
We had commented this out when we implemented "reject crash reports from unsupported products" so we could test the code on stage. This code is here solely so that stage processes all the incoming crash reports sent by prod. This makes it easier to see some things on stage and debug issues with the processor and webapp.

However, we have good tools for taking crash data from prod and submitting it to stage--we don't need a stage-specific exception and this code is funky and creates other issues and prevents us from testing throttling on stage.

So given all that, I figure it's easier maintenance-wise to just ditch it.